src/betterproto/__init__.py

```diff
@@ -1056,6 +1122,9 @@ def delta_to_json(delta: timedelta) -> str:
 
 
 class _Timestamp(Timestamp):
+    seconds: int = int64_field(1)
```

Comment:
They aren't picked up and added to the class's fields iirc?

Reply:
Normally inheritance works with dataclasses

Reply:
Protobufs however don't support this behaviour from the research I've done, if this is actually part of the spec I would consider it but it would slow stuff down.